### ISSUE TITLE
Introduction Package 4.6 not working on cms-base-distribution12

### ISSUE THREAD

**Issue by florinppgmail** [FriendsOfTYPO3/introduction#105]
Typo3 CMS ddev installation:
- ddev composer create "typo3/cms-base-distribution:^12"
Introduction Package extension installation:
- composer req typo3/cms-introduction
- as described in the  official extension page https://extensions.typo3.org/extension/introduction

compose.json shows:
- "typo3/cms-backend": "^12.4.0",
- "typo3/cms-introduction": "^4.6",

Following errors occur in backend:
Web -> List : An exception occurred while executing a query: Table 'db.index_config' doesn't exist
Web -> Indexing : An exception occurred while executing a query: Table 'db.index_phash' doesn't exist
Admin Tools -> Maintainance/Settings/Upgrade/Environment are not displaying anything anymore

**Comment by gilbertsoft** [comment]
Hi @florinppgmail , looks like you missed to run `composer exec typo3 extension:setup` like mentioned in the docs see https://docs.typo3.org/p/typo3/cms-introduction/4.6/en-us/Installation.html

**Comment by florinppgmail** [comment]
Hey @gilbertsoft , thanks for the quick reply.
Indeed I reached that Installation page and saw the second command.

I guess the Extension page is somehow misleading, regarding the installation instructions just with the composer command.
Can someone add the second command in the "Install extension via composer command" tab below in the extension page?

Also, I've tried to follow up some of the instructions from the [Maintainance](https://docs.typo3.org/p/typo3/cms-introduction/4.6/en-us/Maintenance.html) page and it seems there are some issues:
- `ddev reset ` command is not available anymore 
- `ddev render-docs` command is not available
- the docs url https://docs.introduction.<typo3-app-folder>.ddev.local/ is not workin either

In the hope that I'm not doing it wrong somehow, could this be double checked for accuracy?

Thank you for your support Gilbert!

**Comment by gilbertsoft** [comment]
> I guess the Extension page is somehow misleading, regarding the installation instructions just with the composer command. Can someone add the second command in the "Install extension via composer command" tab below in the extension page?

Agree, the instructions on TER could be better for distributions where `typo3 extension:setup` is a must.

I've created an issue for TER see https://git.typo3.org/services/t3o-sites/extensions.typo3.org/ter/-/issues/591

> Also, I've tried to follow up some of the instructions from the [Maintainance](https://docs.typo3.org/p/typo3/cms-introduction/4.6/en-us/Maintenance.html) page and it seems there are some issues:
> 
>     * `ddev reset ` command is not available anymore
> 
>     * `ddev render-docs` command is not available
> 
>     * the docs url https://docs.introduction..ddev.local/ is not workin either

This is related to the maintenance of the Introduction Package and only available if you clone the repository. Please carefully read the box at the top of the page.

**Comment by florinphp** [comment]
That's great. Thank you for the support with updating instructions and the clarifications on the Maintainance page.

**Comment by gilbertsoft** [comment]
Change at TER is done and will be deployed soon.
